src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -363,10 +363,10 @@ export class AppComponent implements OnInit, AfterViewChecked {
                     };
 
                     // Precalculate the max height and min height
-                    let maxHeight = resultSet.totalRows < self._defaultNumShowingRows
-                        ? Math.max((resultSet.totalRows + 1) * self._rowHeight, self.dataIcons.length * 30) + 10
+                    let maxHeight = resultSet.rowCount < self._defaultNumShowingRows
```

Comment:
The lines themselves are covered in the tests (probably due to the fact that these lines are within `ngOnInit()` ), but I don't think any tests specifically verify maxHeight  or anything like that.